README.md

```diff
@@ -102,7 +102,7 @@ Output:
         },
         {
           browser: 'chrome',
-          browser_version: 14.0,
+          browser_version: '27.0 beta',
```

Comment:
Please update the example to current versions

Reply:
updated

Reply:
You can resolve this comment